tools/fiptool/fiptool.c

```diff
@@ -826,7 +826,7 @@ static void create_usage(void)
 	    "pointed to by file.\n");
 	printf("  --plat-toc-flags <value>\t16-bit platform specific flag field "
 	    "occupying bits 32-47 in 64-bit ToC header.\n");
-	fputc('\n', stderr);
+	printf("\n");
```

Comment:
Good catch, generally usage output should all go to stderr but in this case it does not really matter because fiptool cannot be used in a pipeline so I am okay with this.